osmnx/save_load.py

```diff
@@ -531,13 +533,12 @@ def graph_to_gdfs(G, nodes=True, edges=True, node_geometry=True, fill_edge_geome
 
         start_time = time.time()
 
-        nodes = {node:data for node, data in G.nodes(data=True)}
-        gdf_nodes = gpd.GeoDataFrame(nodes).T
+        nodes_data = [data for _, data in G.nodes(data=True)]
+        gdf_nodes = gpd.GeoDataFrame(nodes_data, index=G.nodes(data=False))
```

Comment:
Will `G.nodes` return the elements in the same order each time? If it doesn't, then nodes_data and the index will be misaligned. Per the networkx docs, `G.nodes` returns a dict-like interface, and Python dicts are not ordered: https://networkx.github.io/documentation/stable/reference/classes/generated/networkx.Graph.nodes.html

Reply:
Thanks, now I'm likewise not sure anymore. I will push a fix.

By the way: Happy new year!

Reply:
Which commit resolved this?